qp_qiime2/__init__.py

```diff
@@ -21,176 +25,179 @@
 
 # Define the rarefy command
 req_params = {
-    'i-table': ('artifact', ['BIOM']),
-    'p-sampling-depth': ['integer', 1000]
+    'BIOM table': ('artifact', ['BIOM']),
+    'Sampling depth': ['integer', 1000]
 }
 opt_params = {}
 outputs = {'o-table': 'BIOM'}
 dflt_param_set = {
     'Defaults': {
-        'p-sampling-depth': 1000}
+        'Sampling depth': 1000}
 }
 qiime_cmd = QiitaCommand(
-    "Rarefy", "Rarefy",
+    "Rarefy features", "Rarefy",
     rarefy, req_params, opt_params, outputs, dflt_param_set,
     analysis_only=True)
 plugin.register_command(qiime_cmd)
 
 # Define the beta_diversity command
-req_params = {'i-table': ('artifact', ['BIOM'])}
+req_params = {'BIOM table': ('artifact', ['BIOM'])}
 opt_params = {
-    'p-metric': [
-        ('choice:["sokalsneath", "kulsinski", "mahalanobis", "matching", '
-         '"euclidean", "correlation", "yule", "russellrao", "hamming", '
-         '"jaccard", "braycurtis", "dice", "rogerstanimoto", "sqeuclidean", '
-         '"cityblock", "sokalmichener", "cosine", "wminkowski", "seuclidean", '
-         '"chebyshev", "canberra", "unweighted UniFrac", '
-         '"weighted normalized UniFrac", "weighted unnormalized UniFrac"]'),
-        'jaccard'],
-    'i-tree': ['choice:["default", "None"]', 'None']}
+    'Diversity metric': [
+        'choice:%s' % dumps(list(BETA_DIVERSITY_METRICS)),
+        'Jaccard similarity index'],
+    'Phylogenetic tree': ['choice:["default", "None"]', 'None'],
+    'Number of jobs': ['integer', 1],
+    'Adjust variance (phylogenetic only)': ['boolean', False],
+    'Alpha value (Generalized Unifrac only)': ['float', 0],
+    'Bypass tips (phylogenetic only)': ['boolean', False]}
 outputs = {'distance_matrix': 'distance_matrix'}
```

Comment:
You've renamed the input dictionary keys (e.g. `i-tree` to `Phylogenetic tree`) to be more human-friendly -- should that also be the case for the output dict keys?

Reply:
...see also lines 73, 91, 110, 127, 138, 157, 174, and 192

Reply:
@antgonza @adswafford what do you think?